recipes/mpiFFT4py/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "mpiFFT4py" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/spectralDNS/{{ name }}.git
+  git_tag: {{ version }}
+
+build:
+  skip: true  # [win]
+  number: 1
+
+requirements:
+  build:
```

Comment:
Please add `toolchain`. This sets the deployment target for OS X to 10.9 and also uses `clang` amongst other things.